examples/flutter_view/ios/Runner/MainViewController.m

```diff
@@ -12,9 +12,11 @@ @interface MainViewController ()
 
 @property (strong, nonatomic) NativeViewController* nativeViewController;
 @property (strong, nonatomic) FlutterViewController* flutterViewController;
+@property FlutterMessageChannel* messageChannel;
```

Comment:
You could add (nonatomic) here.  Otherwise the synthesized getter and setter will synchronize access to the property. Strong is default, so there's no need to add that.